package.json

```diff
@@ -53,9 +63,7 @@
     "lint": "run-s lint:editorconfig lint:js",
     "unit": "karma start",
     "test": "run-s lint compile unit",
-    "compile": "npm run compile-es && npm run compile-commonjs",
-    "compile-es": "tsc --outDir es --module es6",
-    "compile-commonjs": "tsc --outDir . --module commonjs",
+    "compile": "tsc --outDir es --module es6",
```

Comment:
If `--outDir` and `--module` are static, then they could be in the `tsconfig.json`.